clients/cobol-lsp-vscode-extension/src/services/settings/LocalCopybookResolver.ts

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2020 Broadcom.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Broadcom, Inc. - initial API and implementation
+ */
+import * as fs from "fs";
+import * as path from "path";
+import {URL} from "url";
+import {CopybookResolver} from "./CopybookResolver";
+import {SettingsUtils} from "./util/SettingsUtils";
+
+/**
+ * This function construct an URI from a valid resource provided from the setting configuration
+ * @param resource represent the file to search within the workspace folder list
+ * @return an URI representation of the file or undefined if not found
+ */
+function getURIFromResource(resource: string): URL {
+    for (const workspaceFolder of SettingsUtils.getWorkspacesURI()) {
+        const uri: URL = new URL(path.join(workspaceFolder, resource));
+        if (fs.existsSync(uri)) {
+            return uri;
+        }
+    }
+}
+
+function resolveURIList(list: string[]): string[] {
+    const result: Set<string> = new Set<string>();
+    list.filter(resource => resource !== "*").forEach(resource => {
```

Comment:
What is "*" for?

Reply:
Represent "all content" and since we discussed and decide to not support at moment this scenario I made an unit test to be sure that user didn't put the star in the path

Reply:
and content is only paths, right?

Reply:
Let's say you have a folder called ```TEST``` that have two file ```ABC``` and ```DEF```.
If the user is tempted to say, I want all the copybooks under test could try to do {"local": ["TEST/*"] but we decided to not support this case at moment, btw it will be pretty straightforward to modify if we'd like to introduce this support